openmc/capi/math.py

```diff
@@ -154,6 +157,27 @@ def calc_zn(n, rho, phi):
     _dll.calc_zn_c(n, rho, phi, zn)
     return zn
 
+def calc_zn_rad(n, rho):
+    """ Calculate the even orders in n-th order modified Zernike polynomial moment with no azimuthal dependency (m=0) for a given radial location in the unit disk. The normalization of the polynomials is such that the integral of Z_pq*Z_pq over the unit disk is exactly pi
```

Comment:
PEP8: Maximum line length should [generally be 80 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).